packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -270,6 +278,46 @@ const Dropdown = React.forwardRef(
     }: DropdownProps<ItemType>,
     ref: ForwardedRef<HTMLButtonElement>
   ) => {
+    const { refs, floatingStyles } = useFloating(
+      autoAlign
+        ? {
+            // placement: direction,
```

Comment:
I think leaving this in will instruct floating-ui to prefer the specified direction where possible
```suggestion
            placement: direction,
```